src/features/metadata-based-view/MetadataBasedItemList.scss

```diff
@@ -14,8 +14,26 @@
     color: $bdl-gray-62;
     border-bottom: 1px solid $bdl-gray-10;
 
-    svg {
-        margin: 0 10px;
+    & .icon-pencil {
+        margin-left: 10px;
+    }
+}
+
+.bdl-MetadataBasedItemList-cell--edit {
+    display: flex;
+    height: 32px;
+    background-color: $white;
+    border: 1px solid $bdl-gray-20;
+    border-radius: 3px;
+
+    .bdl-MetadataBasedItemList-cell--cancelIcon,
+    .bdl-MetadataBasedItemList-cell--saveIcon {
```

Comment:
Since these classes are all modifiers of `.bdl-MetadataBasedItemList-cell`, should they be at the same level, rather than nested?

Reply:
Need this added specificity to get over https://github.com/box/box-ui-elements/blob/master/src/styles/common/_buttons.scss#L91